web_ui/src/world.ts

```diff
@@ -107,5 +107,43 @@ export const Current: World = {
         )
       ).data
     },
+    startTrial: async (args: api.IStartTrialArgs) =>
+      (
+        await authRoute.post<unknown>(
+          `/v1/t/${args.teamId}/start_trial`,
+          jsonToFormData({ billingEmail: args.billingEmail }),
```

Comment:
why `jsonToFormData` and not json?

Reply:
Django supports form data by default and does not support json payloads. I'd need to write a middleware for that.